From 5b08a3855f81330de8e3870f07166fce16daee22 Mon Sep 17 00:00:00 2001
From: Nishanth Menon <nm@ti.com>
Date: Fri, 10 May 2024 11:57:38 -0500
Subject: [PATCH 1/2] ASoC: ti: davinci-mcasp: Lower sample rate warning

When frequency is not exact, the debug information is informative,
but not necessarily informational to the end user. Lower the severity
down.

Fixes: 3e9bee11d831 ("ASoC: davinci-mcasp: Restructure the davinci_mcasp_calc_clk_div()")

Signed-off-by: Nishanth Menon <nm@ti.com>
---
 sound/soc/ti/davinci-mcasp.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sound/soc/ti/davinci-mcasp.c b/sound/soc/ti/davinci-mcasp.c
index 621a9d5f9377..143a3841aa45 100644
--- a/sound/soc/ti/davinci-mcasp.c
+++ b/sound/soc/ti/davinci-mcasp.c
@@ -1155,8 +1155,8 @@ static int davinci_mcasp_calc_clk_div(struct davinci_mcasp *mcasp,
 
 	if (set) {
 		if (error_ppm)
-			dev_info(mcasp->dev, "Sample-rate is off by %d PPM\n",
-				 error_ppm);
+			dev_dbg(mcasp->dev, "Sample-rate is off by %d PPM\n",
+				error_ppm);
 
 		__davinci_mcasp_set_clkdiv(mcasp, MCASP_CLKDIV_BCLK, div, 0);
 		if (reg & AHCLKXE)
-- 
2.51.0

